package.json

```diff
@@ -3,22 +3,22 @@
   "version": "0.1.0",
   "private": true,
   "dependencies": {
-    "axios": "^0.21.1",
-    "i18next": "^21.6.14",
-    "i18next-browser-languagedetector": "^6.1.3",
-    "i18next-http-backend": "^1.4.0",
+    "axios": "^1.4.0",
+    "i18next": "^22.5.1",
+    "i18next-browser-languagedetector": "^7.0.2",
+    "i18next-http-backend": "^2.2.1",
     "react": "^18.2.0",
     "react-dom": "^18.2.0",
-    "react-i18next": "^11.16.1",
+    "react-i18next": "^12.3.1",
     "react-router-dom": "^6.3.0",
     "react-scripts": "5.0.1",
     "sass": "^1.49.11",
-    "web-vitals": "^2.1.4"
+    "web-vitals": "^3.3.2"
```

Comment:
I think we don't need it as removed the `reportWebVitals` entirely, right?

Reply:
Removed ✅